WoWsShipBuilder.UI/UserControls/ShipStatsControl.axaml.cs

```diff
@@ -68,5 +70,17 @@ private async Task OpenGraphsWindow(object sender, PointerReleasedEventArgs e, T
             win.Show((Window)this.GetVisualRoot());
             e.Handled = true;
         }
+
+        public void OpenDepthChargeDamageDistributionChart(object sender, PointerReleasedEventArgs e)
+        {
+            var dc = (ShipStatsControlViewModelBase)DataContext!;
```

Comment:
Avoid non-null assertions
Use ```if (DataContext is ShipStatsControlViewModelBase vm)``` instead or turn the control into a `ReactiveUserControl<ShipStatsControlViewModelBase>`